src/js/rtp-stats.js

```diff
@@ -29,9 +29,9 @@ export function extractAudioStatsFromStats(timestamp, stats, streamType) {
                         if (typeof statsReport.googRtt !== 'undefined') {
                             rttMs = statsReport.googRtt;
                         }
-                        callStats = new AudioRtpStats(timestamp, packetsLost, statsReport.packetsSent, audioLevel, rttMs, null);
+                        callStats = new MediaRtpStats(timestamp, packetsLost, statsReport.packetsSent, audioLevel, rttMs, null);
```

Comment:
Can we get packets count for audio too?

Reply:
connect-rtc supports video media. Does this fix would collect stats for the video as well? Currently there is no video report per se.

Reply:
@yuliangz Yes, there already are both packetCount and packetLost metrics in audio by default. I've test ran locally and it flows normally as expected. 
@antimora Yes, this is to enable us to collect stats for the video by using this web app.

Reply:
Oops, my bad. I meant to ask for bytes count, not packets count.

Reply:
Ok, new version below:

"[{"_timestamp":"2018-01-02T18:31:49.128Z","_packetsLost":"2","_packetsCount":"4095","_audioLevel":"25","_rttMilliseconds":"24","_jbMilliseconds":null,"_bytesSent":"343979"},{"_timestamp":"2018-01-02T18:31:49.129Z","_packetsLost":0,"_packetsCount":"3987","_audioLevel":"60","_rttMilliseconds":null,"_jbMilliseconds":"60","_bytesSent":null,"_bytesReceived":"455774"},{"_timestamp":"2018-01-02T18:31:49.129Z","_packetsLost":"1","_packetsCount":"3957","_audioLevel":null,"_rttMilliseconds":"18","_jbMilliseconds":null,"_bytesSent":"3151752","_bytesReceived":null,"_framesEncoded":"2472","_framesDecoded":null,"_frameRateSent":"29","_frameRateReceived":null},{"_timestamp":"2018-01-02T18:31:49.129Z","_packetsLost":0,"_packetsCount":"3950","_audioLevel":null,"_rttMilliseconds":null,"_jbMilliseconds":"171","_bytesSent":null,"_bytesReceived":"1665803","_framesEncoded":null,"_framesDecoded":"3199","_frameRateSent":null,"_frameRateReceived":"29"}]"